tidb/src/tidb/nemesis.clj

```diff
@@ -29,7 +29,7 @@
                     (:resume-pd :resume-kv :resume-db
                      :start-pd  :start-kv  :start-db) nodes
 
-                    (util/random-nonempty-subset nodes))
+                    (take (condp > (rand) 0.4 1 0.7 2 0.85 3 0.95 4 5) (shuffle nodes)))
```

Comment:
What does the numbers mean?

Reply:
They indicate the probabilities of number of nodes to kill/stop, that is, P(n=1) = 0.4, P(n=2) = 0.3, ...
Previously the number of nodes is chosen between 1~5 uniformly. I found it often make the system unavailable (because a lot of nodes get killed). So I try to make the process nemesis reasonable here.

Reply:
Got, thx. Maybe we could do some parameter tuning on this, too many unavaialble kill will reduce the valid test time.

Reply:
Yes